test/e2e_tests/specs/ClearConversationContent/clearConversationContent.spec.ts

```diff
@@ -0,0 +1,400 @@
+      await expect(userAPages.conversation().messages).toHaveCount(0);
+      await userBPages.conversation().sendMessage('Message from User B after Clear');
+      await userAPages.conversationList().openConversation(userB.fullName);
+      await expect(userAPages.conversation().messages).toHaveCount(1);
+
+      // Step 6.2: Verify you can receive incoming pings
+      await userBPages.conversationList().openConversation(userA.fullName);
+      await userBPages.conversation().sendPing();
+      await userAPages.conversationList().openConversation(userB.fullName);
+      await expect(userAPages.conversation().getPing()).toBeVisible();
+
+      // Step 6.3: Verify you can receive incoming pictures
+      const {page} = userAPages.conversation();
+      await shareAssetHelper(getImageFilePath(), page, page.getByRole('button', {name: 'Add picture'}));
+      await shareAssetHelper(
+        getImageFilePath(),
+        userBPageManager.page,
+        userBPageManager.page.getByRole('button', {name: 'Add picture'}),
+      );
+      const messageWithImage = userAPages.conversation().getMessage({sender: userB}).last();
+      await expect(messageWithImage).toBeVisible();
+
+      // Step 6.4: Verify you can receive incoming files
+      await userAPages.conversationList().openConversation(userB.fullName);
+      await shareAssetHelper(getTextFilePath(), page, page.getByRole('button', {name: 'Add file'}));
+      const messageWithFile = userAPages.conversation().getMessage({sender: userB}).last();
+      await expect(messageWithFile).toBeVisible();
+
+      // Step 6.5: Verify you can receive incoming calls
+      await userAPages.conversationList().openConversation(userB.fullName);
+      await userBPages.conversationList().openConversation(userA.fullName);
+      await userBPages.conversation().startCall();
+      await expect(userAPages.calling().acceptCallButton).toBeVisible();
+    },
+  );
+
+  test(
+    'I want to clear the group conversation content from conversation details options',
+    {tag: ['@TC-424', '@regression']},
+    async ({createPage}) => {
+      const [userAPageManager, userBPageManager] = await Promise.all([
+        PageManager.from(createPage(withLogin(userA), withConnectedUser(userB))),
+        PageManager.from(createPage(withLogin(userB))),
+      ]);
+
+      const {pages: userAPages, modals: userAModals} = userAPageManager.webapp;
+      const userBPages = userBPageManager.webapp.pages;
+
+      const conversationName = 'Group conversation';
+      await createGroup(userAPages, conversationName, [userB]);
+
+      // Step 1: User A and B write in group conversation
+      await userAPages.conversationList().openConversation(conversationName);
+      await userAPages.conversation().sendMessage('Message from User A');
+
+      await userBPages.conversationList().openConversation(conversationName);
+      await userBPages.conversation().sendMessage('Message from User B');
```

Comment:
Just a suggestion, but it might be a good idea to add an assertion before clearing the conversation expecting the number of messages to be 2. Some times such a "counter" assertion can greatly improve confidence in a test :)